INSTALL.txt

```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
     https://github.com/libyal/libewf
     Available at: http://sourceforge.net/projects/libewf/
 
+- LibVHDI: Allows you to process disk images that are stored in the
+Virtual Hard Disk format (VHD).  
+
+    The official repository is available hereL:
+
+    https://github.com/libyal/libvhdi
+
+- LibVMDK: Allows you to process disk images that are stored in the
+VMware Virtual Disk format (VMDK).  
+
+    The official repository is available hereL:
+
+    https://github.com/libyal/libvmdk
+
+- Libvslvm: Allows you to access the Linux Logical Volume Manager (LVM) format 
+that is sotred on a disk image.  Libbfio is a requirement for Libvslvm to work:  
```

Comment:
`Libbfio is a requirement for Libvslvm to work:` maybe change to: `A stand-alone version of libbfio is needed to allow libvslvm to directly read from a TSK_IMAGE`